src/SuperGoodRobot/CLAW_ARM_TEST.ino

```diff
@@ -0,0 +1,24 @@
+void start_claw_arm_test() {
+	LCD.clear();
+	LCD.home();
+	setForearmServo(0);
+	setBasearmServo(0);
+}
+
+void claw_arm_test() {
+	int forearm_position=constrain((byte)map(knob(6),0,KNOB_MAX,0,180),0,180-1); //get the position by mapping the knob
+	int basearm_position=constrain((byte)map(knob(7),0,KNOB_MAX,0,180),0,180-1); //get the position by mapping the knob
```

Comment:
Why map from 0 to 180, and then constrain from 0 to 179?  Isn't this redundant?  Also why exclude 180?